flows/msg.go

```diff
@@ -86,3 +88,31 @@ func (m *MsgIn) SetExternalID(id string) { m.ExternalID_ = id }
 
 // QuickReplies returns the quick replies of this outgoing message
 func (m *MsgOut) QuickReplies() []string { return m.QuickReplies_ }
+
+// Templating returns the templating to use to send this message (if any)
+func (m *MsgOut) Templating() *MsgTemplating { return m.Templating_ }
+
+// MsgTemplating represents any substituted message template that should be applied when sending this message
+type MsgTemplating struct {
+	Template_  assets.TemplateReference `json:"template"`
+	Language_  utils.Language           `json:"language"`
+	Variables_ []string                 `json:"template_variables"`
```

Comment:
this could be just `variables` like the action since it's now inside in a bigger thing called templating

Reply:
Doh ya.. damn renames! Thanks for all the stupid catches.